mod_conf_sql.c

```diff
@@ -157,6 +157,12 @@ static int sqlconf_parse_uri_db(char **uri) {
   *tmp = '\0';
   sqlconf_db.database = pstrdup(sqlconf_pool, *uri);
 
+  int i;
+  int n=strlen(sqlconf_db.server);
+
+  // replace \ with /
+  for(i=0;i<=n;i++) if(sqlconf_db.server[i]=='\\') sqlconf_db.server[i]='/';
```

Comment:
You might be running into shell handling of special characters.  That is, what if you enclosed the entire `-c` argument in single quotes, _e.g._:

    $ proftpd -c 'sql://nouser:nopass@/path/to/sqlitefile.db/db:SQlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap'